src/dmclock_server.h

```diff
@@ -1453,6 +1513,18 @@ namespace crimson {
 #endif
       }
 
+
+      void update_client_info(const C& client_id) {
```

Comment:
This function is repeated for both PullPriorityQueue and PushPriorityQueue. Shouldn't this be a public function in PriorityQueueBase?